### PR TITLE
Add gh-scoped-creds to base-notebook

### PR DESCRIPTION
**Pull request by yuvipanda**
This lets you *safely* push to GitHub from a properly configured GitHub.

See https://blog.jupyter.org/securely-pushing-to-github-from-a-jupyterhub-3ee42dfdc54f
(draft blog post) for more information

### CODE REVIEW

**Comment by yuvipanda** [comment]
@scottyhq i see my pip package doesn't show up in the lock files - does this still work? or only conda packages are installable?

**Comment by yuvipanda** [comment]
https://github.com/conda-forge/staged-recipes/pull/18710 starts the process for bringing this into conda-forge

**Comment by scottyhq** [comment]
thanks @yuvipanda !

since `conda.lock` files do not have pip urls, the pip install happens in a separate step: https://github.com/pangeo-data/pangeo-docker-images/blob/14f8b9f03ee95e7b411a61be6e0ebb4ab978f775/base-image/Dockerfile#L170 

So you'd need to add this package in a separate `requirements.txt` file currently (or add to conda-forge). See also #170 for a possible change to handle everything in a single lockfile

**Comment by yuvipanda** [comment]
@rabernat I think right now there isn't a way to get pip packages installed here in this image.

@scottyhq for this case, perhaps just waiting on https://github.com/conda-forge/staged-recipes/pull/18710 and getting it into conda-forge is the way to go?

**Comment by scottyhq** [comment]
Adding the requirements.txt should work for now, but it won't show up in packages.txt (see #306 ... sorry i linked to the wrong issue above). Longer term, yes, it's best to have everything coming from conda-forge once available.

File: pangeo-notebook/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+gh-scoped-creds==4.0
```
Comment (yuvipanda) [inline pangeo-notebook/requirements.txt:1]:
```suggestion
gh-scoped-creds==4.1
```

**Comment by yuvipanda** [comment]
@scottyhq great! in that case i'm happy for this to be merged :) I can make another PR once this makes it into conda-forge

**Review by scottyhq — APPROVED** [review]

**Pull request merged**